src/setup/devnet/index.js

```diff
@@ -478,9 +478,9 @@ export class Devnet {
                     ];
 
                     // create heimdall folder
-                    shell.exec(`sudo mkdir -p /var/lib/heimdall`)
-                    shell.exec(`sudo chmod 777 -R /var/lib/heimdall/`)
-
+                    shell.exec(`mkdir -p ${this.config.targetDirectory}`)
+                    shell.exec(`chmod 777 -R ${this.config.targetDirectory}`)
+                    console.log('this.config.targetDirectory', this.config.targetDirectory);
```

Comment:
remove log

Reply:
The log i get is 
`this.config.targetDirectory /home/ubuntu/matic-cli/devnet` 
So I guess we need to restore `/var/lib/heimdall`